src/main/java/be/ugent/rml/access/AccessFactory.java

```diff
@@ -259,10 +259,16 @@ private RDBAccess getRDBAccess(QuadStore rmlStore, Term source, Term logicalSour
             password = passwordObject.get(0).getValue();
         }
 
+        String connectionParameters = "";
```

Comment:
Ah interesting! Moving out the connectionParameters helps for performance?

Reply:
It does not – it was moved out do add a bit more felxibility instead of hardcoding modifications to the parameters.  Should probably have been a separate PR.